ci/jenkins/test-vmc.sh

```diff
@@ -629,6 +817,51 @@ function cleanup_cluster() {
     release_static_ip
 }
 
+function cleanup_aws_cluster() {
+    echo "=== Cleaning up AWS cluster ${CLUSTER} ==="
+    export KUBECONFIG=$KUBECONFIG_PATH
+
+    echo "=== Cleaning up Resources ===="
+    export AWS_CONTROLLER_IAM_ROLE=$AWS_SERVICE_USER_ROLE_ARN
+    clusterctl delete --infrastructure aws
+    export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-as-profile --region $AWS_REGION)
+    clusterctl init --infrastructure aws
```

Comment:
Can you explain why it need to run clusterctl delete and init here? Is it because the temporary assumed role has been expired?

Reply:
yes